install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -11,41 +26,14 @@ import (
 // SyndesisSpec defines the desired state of Syndesis
 // +k8s:openapi-gen=true
 type SyndesisSpec struct {
-	// Set RouteHostname to the hostname of the exposed syndesis service.  Typically the operator can automatically
-	// determine this by looking at the result of the Route object it creates.
-	RouteHostname      string `json:"routeHostname,omitempty"`
```

Comment:
Looks like you didn't run `go fmt`

Reply:
Why? I do have it  enabled as File Watcher ... :thinking: